src/store/modules/actions/user.js

```diff
@@ -91,7 +89,7 @@ export async function updatePreferences(
     commit(types.USER_UPDATE_USER_INFO, { creatorPitch });
   }
   if (Object.keys(preferences).length) {
-    if (userStore.getUserInfo)
+    if (getters.getUserInfo)
```

Comment:
I just checked `getUserInfo` is an [object](https://github.com/likecoin/liker-land/blob/61e212716a7333ef9c465da3786804cf45aa246f/src/store/modules/user.js#L14), so it is better to use `getUserId` for the condition